src/System.Runtime.Analyzers/Core/SpecifyCultureInfo.cs

```diff
@@ -24,15 +27,86 @@ public abstract class SpecifyCultureInfoAnalyzer : DiagnosticAnalyzer
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Globalization,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: Uri,
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.RegisterCompilationStartAction(csaContext =>
+            {
+                var cultureInfoType = csaContext.Compilation.GetTypeByMetadataName("System.Globalization.CultureInfo");
+                if (cultureInfoType != null)
+                {
+                    csaContext.RegisterOperationAction(oaContext =>
+                    {
+                        var invocationExpression = (IInvocationExpression)oaContext.Operation;
+                        var targetMethod = invocationExpression.TargetMethod;
+                        if (targetMethod.ContainingType == null)
+                        {
+                            return;
+                        }
+
+                        var methodsWithSameNameAsTargetMethod = targetMethod.ContainingType.GetMembers(targetMethod.Name).OfType<IMethodSymbol>();
```

Comment:
What _is_ our policy on `var`? Would `IEnumerable<IMethodSymbol>` be preferable here?